_common/core_ref.md

```diff
@@ -1,5 +1,21 @@
 ---
-layout: externalurl
-redirect_url: https://core.blockstack.org/
+layout: core
 title: "Stacks Node API"
+description: Interacting with the Blockstack Blockchain
+permalink: /:collection/:path.html
 ---
+
+With the launch of Stacks 2.0, a new version of the Blockstack blockchain was released. You currently have two ways of interacting with the blockchain:
+
+- Using a hosted HTTPS REST client that makes RPC calls to the blockchain
+- Directly making JSON RPC calls to the blockchain yourself
+
+Using the hosted client requires you to trust the client but provides a faster onboarding.
+
+## Hosted REST Client
```

Comment:
I think this should be titled as the sidecar (hopefully until we opt to refer to it as the generic stacks-blockchain-api).